clients/pricing-client/package.json

```
@@ -23,7 +23,7 @@
   "scripts": {
     "test": "jest",
     "bundle-definition": "webpack",
-    "openapi": "node ../../scripts/update-openapi.js /Users/juanjoseguevara/Desktop/code/pricing-api/lambda/ApiHandlerFunction/openapi.yml",
+    "openapi": "node ../../scripts/update-openapi.js https://docs.api.epilot.io/pricing-api.yaml",
```

Could I make a proposal here? Now we're assuming we always want to generate from the already deployed spec, while in some situations we might want to do it from the spec we have locally. Could we add a second script to cover both scenarios? Of course, this second script now makes the assumption we have cloned pricing-api and sdk-js into the same directory.
```suggestion
    "openapi": "node ../../scripts/update-openapi.js https://docs.api.epilot.io/pricing-api.yaml",
    "openapi:local": "node ../../scripts/update-openapi.js ../../../pricing-api/lambda/ApiHandlerFunction/openapi.yml",
```

---

I think this is a great idea